.gitignore

```diff
@@ -6,3 +6,5 @@ compile_commands.json
 .build
 .compile_commands.json
 .compile_commands.tmp
+.vscode
+.DS_Store
```

Comment:
The TherapSID.zip file does not belong into the repo.
Please delete it using `git rm TherapSID.zip` (you might want to add `*.zip` to the `.gitignore` also), then commit and push.